mail/urls.py

```diff
@@ -16,4 +16,9 @@
     url(r'^api/v0/mail/course/(?P<course_id>[\d]+)/$', CourseTeamMailView.as_view(), name='course_team_mail_api'),
     url(r'^api/v0/mail/learner/(?P<student_id>[\d]+)/$', LearnerMailView.as_view(), name='learner_mail_api'),
     url(r'^api/v0/mail/automatic_email/$', AutomaticEmailView.as_view(), name='automatic_email_api'),
+    url(
```

Comment:
@aliceriot DRF appends suffixes to the urls, depending on which one you need: http://www.django-rest-framework.org/api-guide/routers/#defaultrouter